kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```diff
@@ -93,45 +108,610 @@
+      "required": ["internId","navn"],
+      "additionalProperties": false
+    },
+
+    "kjoenn": {
+      "description": "Samme enum som folkeregisteret",
+      "type": "string", "enum": ["KVINNE","MANN"]},
+
+    "personForetak": {
+      "description": "En siktet eller fornærmet kan være en person eller et foretak, denne typen er enten person eller foretak.",
+      "type": "object",
+      "properties": {
+        "type": {"enum": ["person", "foretak"]},
+        "person": {"$ref": "#/definitions/person"},
+        "foretak": {"$ref": "#/definitions/foretak"}
+      },
+      "required": ["type"],
+      "oneOf": [
+        {
+          "properties":{
+            "type": {"const": "person" },
+            "foretak": {"type": "null"}
+          },
+          "required": ["person"]
+        },
+        {
+          "properties":{
+            "type": {"const": "foretak" },
+            "person": {"type": "null"}
+          },
+          "required": ["foretak"]
+        }
+      ]
+    },
+
+    "personForetakEnkel": {
+      "description": "En siktet eller fornærmet kan være en person eller et foretak, denne typen er enten person eller foretak.",
+      "type": "object",
+      "properties": {
+        "type": {"enum": ["personEnkel", "foretakEnkel"]},
+        "personEnkel": {"$ref": "#/definitions/personEnkel"},
+        "foretakEnkel": {"$ref": "#/definitions/foretak"}
+      },
+      "required": ["type"],
+      "oneOf": [
+        {
+          "properties":{
+            "type": {"const": "personEnkel" },
+            "foretakEnkel": {"type": "null"}
+          },
+          "required": ["personEnkel"]
+        },
+        {
+          "properties":{
+            "type": {"const": "foretakEnkel" },
+            "personEnkel": {"type": "null"}
+          },
+          "required": ["foretakEnkel"]
+        }
+      ]
+    },
+
+    "personAdresse": {
+      "description": "Kan være graderte adresser",
+      "type": "object",
+      "properties": {
+        "gradering": {"$ref": "#/definitions/adresseGradering"},
+        "adresse": {"$ref": "#/definitions/adresse"}
+      },
+      "required": ["adresse"],
+      "additionalProperties": false
+    },
+   
+    "person": {
+      "description": "Person med alle nyttige data som fødselsnummer adresse osv., SSP nummer kun på siktede, tiltalte.",
+      "type": "object",
+      "properties": {
+        "internId": {"type": "string", "description": "Intern id i melding, samme person i denne meldingen vil ha samme internId"},
+        "navn": {"$ref": "#/definitions/personnavn"},
+        "foedselsdato": {"type": "string", "format": "date"},
+        "kjoenn": {"$ref": "#/definitions/kjoenn", "description": "Ukjent kjønn hvis denne ikke er med"},
+        "statsborgerskap": {"type": "array", "items": {"$ref": "#/definitions/land"}},
+        "identitetsnummer": {"$ref": "#/definitions/personIdentifikator", "description": "Fødselsnummer, D-nummer eller SSP nummer som er den i bruk"},
```

Comment:
Skal vel og være required(?)

Reply:
Bruker samme modell også på vitner og fornærmede, lager ikke egen definisjon for siktede.